### ISSUE TITLE
System.InvalidCastException

### ISSUE THREAD

**Issue by andwah** [Starcounter/Starcounter#2514]
Start the launcher and navigate to
```
http://localhost:8080/launcher/workspace/abc
```
```
System.InvalidCastException: Unable to cast object of type 'System.Byte[]' to type 'Starcounter.Json'.
   at Starcounter.Response.GetContent[T]() in c:\github\Level1\src\Starcounter.Internal\Http\Response.cs:line 541
   at Starcounter.X.GET[T](String uri, Int32 receiveTimeoutMs, HandlerOptions ho) in c:\github\Level1\src\Starcounter.Node\X.cs:line 219
   at Launcher.WorkspaceResponse(String appName, String uri) in c:\github\Polyjuice\Launcher\Server\Launcher.json.cs:line 209
   at Launcher.<Main>b__9(String appName) in c:\github\Polyjuice\Launcher\Server\Launcher.json.cs:line 125
   at lambda_method(Closure , Request , IntPtr , IntPtr )
   at Starcounter.Rest.UserHandlerInfo.RunUserDelegates(Request req, IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, HandlerOptions handlerOptions) in c:\github\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 230
   at Starcounter.Internal.Web.AppRestServer.RunDelegateAndProcessResponse(IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, Request req, HandlerOptions handlerOptions) in c:\github\Level1\src\Starcounter.Apps.JsonPatch\AppRestServer.cs:line 139
   at Starcounter.Internal.AppsBootstrapper.ProcessExternalRequest(Request req) in c:\github\Level1\src\Starcounter.Apps.JsonPatch\AppsBootstrapper.cs:line 223
HResult=-2147467262

More information about this error may be available in the server error log.
```

**Comment by alemoi** [comment]
Fixed via 6aed03b2ca086fc7708d0b13760a738fb821dfe4
Thank you for reporting!

**Comment by dan31** [comment]
@alemoi, any follow-up test possible to add?
